src/odemis/gui/cont/tabs/fibsem_tab.py

```
@@ -23,6 +23,8 @@
 import logging
 import math
 from typing import Dict
+import numpy
+from concurrent.futures import CancelledError
 
 import numpy
```

Ruff (0.14.14)

[error] Redefinition of unused `numpy` from line 26: `numpy` redefined here

Remove definition: `numpy`

(F811)